docs/source/index.rst

```diff
@@ -56,7 +56,6 @@ Daml Documentation
    :caption: Operating Daml
 
    operating-daml
-   ops/requirements
```

Comment:
Please take a look at https://github.com/digital-asset/daml/blob/main/docs/README.md#updating-the-table-of-contents. We need to mirror this change for the pdf index as well as the index files in the assembly repo.

Reply:
`docs/configs/pdf/index.rst` updted.

Reply:
I think you forgot to push.